ui/app/pages/first-time-flow/index.scss

```diff
@@ -148,10 +149,8 @@
   }
 
   &__checkbox-label {
-    font-style: normal;
-    font-weight: normal;
-    line-height: normal;
-    font-size: 18px;
+    @include H4;
```

Comment:
This change in `line-height` seems fine - it appears to have no affect in practice.